### PR TITLE
flip logic for /available/ disks filtering

### LINKED ISSUE #1721: cephdisks.unused falsely returns mounted disks
### Description of Issue/Question

```
Device Path               Size         rotates available Model name
/dev/vda                  20.00 GB     True    False
/dev/vdb                  20.00 GB     True    False
/dev/vdc                  20.00 GB     True    False
/dev/vdd                  20.00 GB     True    False
/dev/vde                  20.00 GB     True    False
/dev/vdf                  20.00 GB     True    False
/dev/vdg                  10.00 GB     True    False
/dev/vdh                  10.00 GB     True    False

data1:~ # lsblk
NAME                                                                                      MAJ:MIN RM SIZE RO TYPE MOUNTPOINT
vda                                                                                       254:0    0  20G  0 disk
└─vda1                                                                                    254:1    0  20G  0 part /
vdb                                                                                       254:16   0  20G  0 disk /mnt


admin:~ # salt-run disks.c_v_commands
Found DriveGroup <default>
Calling dg.c_v_commands on compound target I@roles:storage
|_
  ----------
  data1:
      - ceph-volume lvm batch --no-auto /dev/vdb --yes


```

it's not used by ceph but it's mounted and therefore not usable.

### PR DESCRIPTION
**Pull request by jschmid1**
Signed-off-by: Joshua Schmid <jschmid@suse.de>

Fixes #1721

-----------------

**Checklist:**
- [x] Added unittests and or functional tests
~- [ ] Adapted documentation~
- [x] Referenced issues or internal bugtracker
- [ ] Ran integration tests successfully (trigger with "@susebot run teuthology" in a GitHub comment; see the [wiki](https://github.com/SUSE/DeepSea/wiki/Testing) for more information)

### CODE REVIEW

**Comment by jschmid1** [comment]
@susebot run teuthology

File: srv/salt/_modules/cephdisks.py [hunk 1]
```diff
@@ -47,9 +47,9 @@ def __init__(self, **kwargs) -> None:
         self.root_disk = self._find_root_disk()
 
     @property
-    def exclude_available(self) -> bool:
+    def exclude_unavailable(self) -> bool:
         """ The available filter """
```
Comment (shyukri) [inline srv/salt/_modules/cephdisks.py]: maybe change the doc string here?

Comment (jschmid1) [reply srv/salt/_modules/cephdisks.py]: done

**Comment by jschmid1** [comment]
that's new @shyukri 

```
2019-08-13T09:53:14.313 INFO:teuthology.orchestra.run.target192168000147:Running: 'sudo mkfs.ext4 -F /dev/vdd; sudo mount /dev/vdd /mnt'
2019-08-13T09:53:14.533 INFO:teuthology.orchestra.run.target192168000147.stderr:mke2fs 1.43.8 (1-Jan-2018)
2019-08-13T09:53:14.535 INFO:teuthology.orchestra.run.target192168000147.stderr:/dev/vdd is apparently in use by the system; will not make a filesystem here!
2019-08-13T09:53:14.614 INFO:teuthology.orchestra.run.target192168000147.stderr:mount: /mnt: unknown filesystem type 'LVM2_member'.
```

**Comment by jschmid1** [comment]
dashboard e2e tests are ok to fail currently (I guess)

**Comment by shyukri** [comment]
> that's new @shyukri
> 
> ```
> 2019-08-13T09:53:14.313 INFO:teuthology.orchestra.run.target192168000147:Running: 'sudo mkfs.ext4 -F /dev/vdd; sudo mount /dev/vdd /mnt'
> 2019-08-13T09:53:14.533 INFO:teuthology.orchestra.run.target192168000147.stderr:mke2fs 1.43.8 (1-Jan-2018)
> 2019-08-13T09:53:14.535 INFO:teuthology.orchestra.run.target192168000147.stderr:/dev/vdd is apparently in use by the system; will not make a filesystem here!
> 2019-08-13T09:53:14.614 INFO:teuthology.orchestra.run.target192168000147.stderr:mount: /mnt: unknown filesystem type 'LVM2_member'.
> ```
> 
> maybe add a `lvremove/vgremove/pvremove` or even a `sgdisk -Z` before?

I've not seen it on https://github.com/SUSE/ceph/pull/322

**Comment by jschmid1** [comment]
@susebot run teuthology

**Comment by kshtsk** [comment]
@susebot run teuthology using wip-qa-drive-replace-fix from https://github.com/shyukri/ceph

**Comment by shyukri** [comment]
Only the dashboard tests are failing, Drive replacement one lists correctly only one device as expected:
```
2019-08-14T15:59:44.379 INFO:teuthology.orchestra.run.target192168000115.stdout:Device Path               Size         rotates available Model name
2019-08-14T15:59:44.379 INFO:teuthology.orchestra.run.target192168000115.stdout:/dev/vdc                  10.00 GB     True    True
2019-08-14T15:59:44.379 INFO:teuthology.orchestra.run.target192168000115.stdout:/dev/vda                  40.00 GB     True    False

....
sudo salt-run disks.c_v_commands
...
ceph-volume lvm batch --no-auto /dev/vdc --yes --osd-ids 2
```

**Comment by shyukri** [comment]
@susebot run teuthology for suite deepsea:tier2:drive-replacement using wip-qa-drive-replace-fix from https://github.com/shyukri/ceph

**Pull request merged**